### DISCUSSION TITLE
Can anyone tell me does this work on a apple M1 locally......if yes how to run it ???

### DISCUSSION THREAD

**Discussion by NitishN31** [lllyasviel/Fooocus#1173] [Q&A]
I have apple m1 macbook and i was curious about how would it perform for image generation and all. I have used SDXL and comfyui and that was not so good and not so bad experiece.    

If anyone know anything about this topic please reply.......

**Comment by mel870** [comment]
I have an M1 Max and while it's useable, it's incredibly slow like Comfy and Auto1111. With new Turbo models, you may have faster performance if you don't need to use any other SDXL or 1.5 refiner models.  With that said, it's best to run Fooocus on a rented GPU or Colab for best performance until it's better optimized for Mac. See options in the following thread: 

https://github.com/lllyasviel/Fooocus/discussions/1155

**Reply by jtgsystems** [reply]
and u can adjust the python files in the config, and the ui to make the outputs up to 5000  - note the computer at most would do 250-300. put i had mine do 200 without issues, 2080 super. 32GB ram.

**Reply by jtgsystems** [reply]
co lab might be able to do more i haven't tried it out yet.  but youd have a machine all day saving the best images, next im trying to mess with the resolutions output to get 4k out of it.
